pkg/apply/applier_test.go

```diff
@@ -1949,7 +1953,8 @@ func TestReadAndPrepareObjectsNilInv(t *testing.T) {
 
 func TestReadAndPrepareObjects(t *testing.T) {
 	inventoryObj := testutil.Unstructured(t, resources["inventory"])
-	inventory := inventory.WrapInventoryInfoObj(inventoryObj)
+	invInfo, err := inventory.ConfigMapToInventoryInfo(inventoryObj)
+	assert.NoError(t, err)
```

Comment:
require maybe? prolly want to skip the rest of the tests if it errors

Reply:
Done